docs/conf.py

```diff
@@ -101,8 +101,7 @@
 html_theme_path = [mdn_theme.get_theme_dir()]
 
 # Theme options are theme-specific and customize the look and feel of a theme
-# further.  For a list of options available for each theme, see the
-# documentation.
+# further. For a list of options available for each theme, see the documentation.
```

Comment:
Should be fixed [upstream](https://github.com/sphinx-doc/sphinx/blob/f3118e1cd6ecb428b0dc56300e2913fd86ccefa3/sphinx/templates/quickstart/conf.py_t#L100), not here.